bootstrap.php

```diff
@@ -70,6 +70,12 @@
     ->setClass(SingleSiteSectionProvider::class)
     ->setShared(true)
 
+    // Site applications
+    ->rule(\Vanilla\Contracts\Site\ApplicationProviderInterface::class)
+    ->setClass(\Vanilla\Site\ApplicationProvider::class)
+    ->addCall('add', [new \Vanilla\Site\Application('garden', ['api', 'entry', 'sso', 'utility'])])
```

Comment:
Normally when initializing container items we do them lazily by using `\Garden\Container\Reference`.

```suggestion
    ->addCall('add', [new \Garden\Container\Reference(
	    \Vanilla\Site\Application::class,
	    ['garden', ['api', 'entry', 'sso', 'utility']]
    )])
```